bangazonapi/views/profile.py

```diff
@@ -177,15 +177,15 @@ def cart(self, request):
             try:
                 open_order = Order.objects.get(
                     customer=current_user, payment_type=None)
-                line_items = OrderProduct.objects.filter(order=open_order)
-                line_items = LineItemSerializer(
-                    line_items, many=True, context={'request': request})
+                lineitems = OrderProduct.objects.filter(order=open_order)
+                lineitems = LineItemSerializer(
+                    lineitems, many=True, context={'request': request})
 
                 cart = {}
                 cart["order"] = OrderSerializer(open_order, many=False, context={
                                                 'request': request}).data
-                cart["order"]["line_items"] = line_items.data
-                cart["order"]["size"] = len(line_items.data)
+                cart["order"]["lineitems"] = lineitems.data
```

Comment:
`lineitems` is already being added to the order in the serializer https://github.com/nss-day-cohort-48/bangazon-S-L-Murphey/blob/main/bangazonapi/views/order.py#L30
This line just needs to be deleted to complete the ticket

Reply:
New commit pushed up without that line.